agent-bootstrap/src/main/java/io/opentelemetry/auto/bootstrap/instrumentation/decorator/BaseDecorator.java

```diff
@@ -68,19 +68,26 @@ public Span onError(final Span span, final Throwable throwable) {
   public Span onPeerConnection(final Span span, final InetSocketAddress remoteConnection) {
     assert span != null;
     if (remoteConnection != null) {
-      onPeerConnection(span, remoteConnection.getAddress());
-
+      InetAddress remoteAddress = remoteConnection.getAddress();
```

Comment:
@trask Just confirming we should use final with local vars? I personally don't like it much but happy to follow the style

Reply:
you are in good company 😄, we recently decided against it, just opened #592 to track updating existing style